modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -501,6 +504,42 @@ namespace core {
             return in.withType(in.depth, in.chan).withSize(dsize);
         }
     };
+
+    G_TYPED_KERNEL(GParseSSDWL, <GDetections(GMat, GOpaque<Size>, float, int)>, "org.opencv.core.parseSSDWL") {
```

Comment:
I'd recommend to rename (at least the string part) to `"org.opencv.core.parseSSD_WL"` for a better readability. What is WL here?

Reply:
Fixed. WL - with labels. It is for splitting kernels (ParseSSD) with different outputs. Changed to BL - boxes and labels.